pkg/services/ngalert/api/tooling/api.json

```diff
@@ -755,7 +758,7 @@
      "type": "string"
     },
     "ErrorSource": {
-     "$ref": "#/definitions/ErrorSource"
+     "$ref": "#/definitions/Source"
```

Comment:
ErrorSource is defined as `type ErrorSource = status.Source`: https://github.com/grafana/grafana-plugin-sdk-go/blob/66c983fae2ae10f731b84c5368a7ca7227bf4f31/backend/error_source.go#L11